events/s3_object_lambda.go

```diff
@@ -0,0 +1,64 @@
+package events
+
+type S3ObjectLambdaEvent struct {
+	XAmzRequestID        string                              `json:"xAmzRequestId"`
+	GetObjectContext     *S3ObjectLambdaGetObjectContext     `json:"getObjectContext,omitempty"`
+	ListObjectsContext   *S3ObjectLambdaListObjectsContext   `json:"listObjectsContext,omitempty"`
+	ListObjectsV2Context *S3ObjectLambdaListObjectsV2Context `json:"listObjectsV2Context,omitempty"`
+	HeadObjectContext    *S3ObjectLambdaHeadObjectContext    `json:"headObjectContext,omitempty"`
+	Configuration        S3ObjectLambdaConfiguration         `json:"configuration"`
+	UserRequest          S3ObjectLambdaUserRequest           `json:"userRequest"`
+	UserIdentity         S3ObjectLambdaUserIdentity          `json:"userIdentity"`
+	ProtocolVersion      string                              `json:"protocolVersion"`
+}
+
+type S3ObjectLambdaGetObjectContext struct {
+	InputS3Url  string `json:"inputS3Url"`
```

Comment:
Linter should have flagged this 🧐 Should be `InputS3URL` instead of `InputS3Url`

Reply:
Thanks, I fixed them.

Reply:
Thanks for letting me know!
I'll create another pull request for that.